.github/workflows/actions.yml

```diff
@@ -60,5 +60,5 @@ jobs:
         POSTGRES_DB: test
         OSMCHA_API_KEY: ${{ secrets.OSMCHA_API_KEY }}
       run: |
-        docker-compose run mapswipe_workers_creation python -m unittest discover --verbose --start-directory tests/unittests/
-        docker-compose run mapswipe_workers_creation python -m unittest discover --verbose --start-directory tests/integration/
\ No newline at end of file
+        docker-compose run --rm mapswipe_workers_creation python -m unittest discover --verbose --start-directory tests/unittests/
+        docker-compose run --rm mapswipe_workers_creation bash -c 'pip install pytest && py.test -ra -v --durations=10 tests/integration/'
```

Comment:
I missed the change notice. Will use pytest.

I think using the latest pytest shouldn't have any impact as there is not much here in the code related to it. But if you think otherwise let us know.
Let's discuss further about dependency management for mapswipe_workers.